libs/features/personalization/personalization.js

```diff
@@ -271,30 +272,91 @@ function parsePlaceholders(placeholders, config, selectedVariantName = '') {
   return config;
 }
 
-function getPersonalizationVariant(manifestPath, variantNames = [], variantLabel = null) {
+export const loadEntitlements = (() => {
+  let entitlements;
+  return (async () => {
+    if (!entitlements) {
+      const { default: getUserEntitlements } = await import('../../blocks/global-navigation/utilities/getUserEntitlements.js');
+      await loadIms();
```

Comment:
Would it bring any value to have these coupled together in a `Promise.all`?

Reply:
👍 Good idea.  Really ims should already have been (or started) loading, this just makes sure before firing getUserEntitlements.